utils/utils.go

```diff
@@ -140,3 +142,16 @@ func ProgressBar(prefix string, size int64, onComplete string) (*mpb.Progress, *
 
 	return p, bar
 }
+
+// CheckMaxMemory gets the total system memory and compares it to the variable.  if the variable
+// is larger than the total memory, it returns an error
+func CheckMaxMemory(newMem strongunits.MiB) error {
```

Comment:
Can you move this somehwere under pkg/machine?
I dislike all these random util/utils packages in our repos.